### PR TITLE
fix(DB) Scout Jyoba should be dead

### PR DESCRIPTION
**Pull request by M4th3m4tic4l**
## Changes Proposed:
-  Update Unit Flag to make it non-attackable

## Issues Addressed:

- https://github.com/azerothcore/azerothcore-wotlk/issues/14295

## Tests Performed:

- 
WoWScrnShot_012923_200814

## How to Test the Changes:

1. Insert the update in your World database
2. Be alliance
3. .go c 84372
4. Attack him

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
unit_flags are sniffed fields, but I have an idea what the fix would be, just a sec while I parse a packet.

**Comment by Gultask** [comment]
Type Flags: 128 (InteractWhileDead)
[1] Position: X: 719.49133 Y: 5415.5103 Z: -26.054508
[1] Orientation: 1.151917338371276855
[1] MovementFlags: 1024 (Root)
[1] [0] NpcFlags: 2
[1] Flags: 537133824 (IMMUNE_TO_PC, IMMUNE_TO_NPC, STUNNED, PREVENT_EMOTES_FROM_CHAT_TEXT) - STUNNED is given from the aura
[1] Flags2: 2049

Aura 29266 (Permanent Feign Death)

The main thing here is the aura.

**Comment by Gultask** [comment]
@M4th3m4tic4l 
Set bytes1&2 in template_addon to 0 and auras to 29266
Also remove STUNNED and FEIGN_DEATH from unit_flags

File: data/sql/updates/pending_db_world/rev_1675054031508722800.sql [hunk 1]
```diff
@@ -1,3 +1,5 @@
 --
 
-UPDATE `creature_template` SET `unit_flags` = 537133824, `unit_flags2` = 2049 WHERE `entry` = 18035;
+UPDATE `creature_template` SET `unit_flags` = 536871680, `unit_flags2` = 2048 WHERE `entry` = 18035;
+
+UPDATE `creature_template_addon` SET `bytes1` =0, `bytes2` =0, `auras` = 29266 WHERE `Entry` = 18035;
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1675054031508722800.sql:5]:
```suggestion
UPDATE `creature_template` SET `unit_flags`=`unit_flags`|536871680, `unit_flags2`=`unit_flags2`|2048 WHERE `entry` = 18035;

UPDATE `creature_template_addon` SET `bytes1` =0, `bytes2` =0, `auras` = '29266' WHERE `Entry` = 18035;
```

**Review by Gultask — COMMENTED** [review]
Just a few changes to follow bitmask standards, and the auras field is technically a comma-separated string 😛 

Leaving for another to review as well since I never personally saw UNIT_FLAG_PREVENT_EMOTES_FROM_CHAT_TEXT before and I don't know if it's added due to aura or it's supposed to be there

**Comment by Maelthyrr** [comment]
Tested also and work. Thanks :)

**Review by Gultask — APPROVED** [review]

**Pull request merged**